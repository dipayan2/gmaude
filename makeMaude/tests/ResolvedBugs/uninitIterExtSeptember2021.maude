set show timing off .
set show advisories off .

***
***	Test for bug in handling of extension information for iter symbols.
***

mod UNINITIALIZED-BUG is
  protecting NAT .
  vars N M : Nat .
  crl [rwc] : s N => M if N => M [nonexec] .
endm

srew 3 using rwc{idle} .
