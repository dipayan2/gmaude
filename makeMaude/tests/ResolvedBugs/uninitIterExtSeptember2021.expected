==========================================
srewrite in UNINITIALIZED-BUG : 3 using rwc{idle} .

Solution 1
rewrites: 1
result NzNat: 2

No more solutions.
rewrites: 3
Bye.
